lib/oli_web/live/workspaces/utils.ex

```diff
@@ -117,7 +117,7 @@ defmodule OliWeb.Workspace.Utils do
   attr :is_active, :boolean, default: false
   attr :sidebar_expanded, :boolean, default: true
   attr :badge, :integer, default: nil
-  attr :on_active_bg, :string, default: "bg-zinc-400 bg-opacity-20"
+  attr :on_active_bg, :string, default: "bg-[#E6E9F2] dark:bg-[#222126]"
```

Comment:
We don't need to address it here, but looking forward we want to build out any of the design tokens used in the designs as tailwind theme tokens and use them here instead of hard-coded values